src/Tools/MicroBuild/run-pdbgit.ps1

```diff
@@ -0,0 +1,70 @@
+<#
+
+    Run PDB Git on our PDB files to enable proper source server support
+
+#>
+Param(
+    [string]$sourcePath = $null,
+    [string]$binariesPath = $null
+)
+
+set-strictmode -version 2.0
+$ErrorActionPreference="Stop"
+
+function Get-PackagesPath {
+    $packagesPath = $env:NUGET_PACKAGES
+    if ($packagesPath -eq $null) {
+        $packagesPath = join-path $env:UserProfile ".nuget\packages\"
+    }
+
+    return $packagesPath
+}
+
+try
+{
+    [xml]$deps = get-content (join-path $sourcePath "build\Targets\Dependencies.props")
+    $pdbGitVersion = $deps.Project.PropertyGroup.PdbGitVersion
+    $packagesPath = Get-PackagesPath
+    $pdbGit = join-path $packagesPath "pdbgit\$pdbGitVersion\tools\PdbGit.exe"
+
+    $signToolDataPath = join-path $sourcePath "build\config\SignToolData.json"
+    $signToolData = convertfrom-json (get-content $signToolDataPath -raw)
+    $allGood = $true
+    $count = 0
+
+    foreach ($relativePath in $signToolData.sign | %{ $_.values }) {
+        $binaryPath = join-Path $binariesPath $relativePath
+        $pdbPath = [IO.Path]::ChangeExtension($binaryPath, ".pdb")
+        if (-not (test-path $pdbPath)) {
+            continue;
+        }
+
+        write-host "Running pdbgit on $pdbPath"
+        & $pdbGit $pdbPath -u https://github.com/dotnet/roslyn --baseDir $sourcePath 
```

Comment:
It needs to be a recognizable name for the git repo.  The tool doesn't actually use this name directly but instead uses the name to determine the repo involved.  It writes a different URL to the PDB.  Generally in the form of `https://raw.github.com/repo/project/<commit>/...`

I based this argument off of the sample on the [PdbGit README](https://github.com/AArnott/PdbGit) file.

```
PdbGit.exe path-to-your.pdb -u https://github.com/username/project --baseDir C:\root-of-your-project\ --commit your-commit-ID
```

We need to be explicit here because in the case of mulitple remotes the tool will pick one of them.  When I ran locally it was actually embedding urls to my personal fork vs. the official once hence I made it an explicit argument.